### DISCUSSION TITLE
Feature request: let us choose the home page/start page

### DISCUSSION THREAD

**Discussion by dasacom** [javikr/sofatime#22] [Ideas]
Hi there,
I opened a feature request.

i think it would be nice to choose the starting page.

There could be a setting option to choose between: discover, my lists, calendar and profile

So i would love to set the app to start with "my lists"

What do you think about it?

best regards, dasacom

**Comment by javikr** [comment]
Hi!
Please note that actually the app starts at the last page the user was. I mean: if the user close the app in the "my lists" page, the app will start next time in "my lists" page.

Anyway, I think it could be a good idea to let the user to reorder the main page items (discover, my lists, calendar and profile). Maybe creating a new screen where users can drag-and-drop as they can do with their lists right now, so they can put "my lists", for example, to the first position.

I'm going to create an issue from this topic and remove the other issue, ok?

Best regards,

**Comment by dasacom** [comment]
Sure 👍
Don’t know why, but this works sometimes AND sometimes not. 
Often starting with discover page.

**Reply by javikr** [reply]
ok! In this case we need to investigate why sometimes don't work. I will open a bug.

Thanks!
